src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -1556,3 +1556,13 @@ def progress_callback(acl_changes):
         failed_entries.append(acl_changes.batch_failures)
 
     namespace.progress_hook = progress_callback
+
+
+def get_not_none_validator(attribute_name):
+    def validate_not_none(cmd, namespace):
+        attribute = getattr(namespace, attribute_name, None)
+        options_list = cmd.arguments[attribute_name].type.settings.get('options_list')
+        if not attribute:
+            from azure.cli.core.azclierror import InvalidArgumentValueError
+            raise InvalidArgumentValueError('argument {} can not be None'.format('/'.join(options_list)))
```

Comment:
Use capital case ahead of an error message.

Reply:
Is this case due to users not specifying the parameter? If so, I think we should use something like `Argument {} should be specified.` directly.

`argument {} can not be None` may be a little ambiguous, making users feel they had provided a `None` value for the parameter.

Reply:
Good suggestion.